plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -751,8 +759,171 @@ bool MeshImportContext::applyEdgeCreases()
   return false;
 }
 
+
+void MeshImportContext::applyColourSetData()
+{
+  const std::vector<UsdGeomPrimvar> primvars = mesh.GetPrimvars();
+  for (auto it = primvars.begin(), end = primvars.end(); it != end; ++it)
+  {
+
+    const UsdGeomPrimvar& primvar = *it;
+    TfToken name, interpolation;
```

Comment:
```
const std::vector<UsdGeomPrimvar> primvars = mesh.GetPrimvars();
for (auto it = primvars.begin(), end = primvars.end(); it != end; ++it)
{
    const UsdGeomPrimvar& primvar = *it;
}
```
Above code can be refactored out to something a bit nicer:
```
for (const UsdGeomPrimvar& primvar : mesh.GetPrimvars())
{
}
or even better with auto
for (const auto& primvar : mesh.GetPrimvars())
{
}
```

Reply:
I'll make this change - but really this is just sugar IMO....in this case I was refactoring existing code and therefore adhering to the existing style.
If it's covered by our upcoming style guide, perfectly happy to adhere to it, but for now I think we should avoid too much stylistic quibbling.